include/psp2kern/kernel/sysmem.h

```diff
@@ -130,7 +130,7 @@ typedef struct SceClass {
 	struct SceClass *uidclass;
 	unsigned int attributes;
 	unsigned short itemsize;
-	unsigned short unk1A;
+	unsigned short itemSizeAligned;
```

Comment:
```suggestion
	unsigned short aligned_itemsize;
```
or `itemsize_aligned`

Reply:
Updated.